### PR TITLE
Add examples of AeDropdown to examples of AeCard

### PR DESCRIPTION
**Pull request by sadiqevani**
This fixes an update on the design where we're displaying the dropdown and not amount in AE.

### CODE REVIEW

File: src/components/ae-icon/ae-icon.md [hunk 1]
```diff
@@ -43,6 +43,7 @@
     <ae-icon fill="secondary" name="back" />
     <ae-icon fill="neutral" name="active" />
     <ae-icon fill="alternative" name="copy" />
+    <ae-icon fill="white" name="copy" />
```
Comment (davidyuk) [inline src/components/ae-icon/ae-icon.md]: This icon is not visible in docs because it's white.

File: src/components/ae-card/ae-card.md [hunk 2]
```diff
@@ -38,11 +58,31 @@
...
+          <ae-icon name="eye" />
+          Details
+        </ae-button>
+      </li>
+    </ae-dropdown>
   </template>
   <ae-text face="uppercase-xs" weight=600 style="margin: 0">Normal Secured</ae-text>
   <ae-address value="ak$QY8VNEkhj7omMUjAvfVBq2NjTDy895LBYbk7qVxQo1qT8VqfE" length="medium" gap=0 />
-  <ae-toolbar fill="primary" align="right" slot="footer">
+  <ae-toolbar fill="primary" fill="white" align="right" slot="footer">
```
Comment (davidyuk) [inline src/components/ae-card/ae-card.md]: It looks a little bit strange:
screenshot from 2018-10-09 19-17-13

File: src/components/ae-card/ae-card.md [hunk 3]
```diff
@@ -10,7 +10,27 @@
...
+          Share Address
+        </ae-button>
+      </li>
+      <li>
+        <ae-button>
+          <ae-icon name="eye" />
+          Details
+        </ae-button>
+      </li>
+    </ae-dropdown>
```
Comment (davidyuk) [inline src/components/ae-card/ae-card.md]: Leave just an AeIcon, examples of AeDropdown should be in the documentation for AeDropdown component.

Comment (sadiqevani) [reply src/components/ae-card/ae-card.md]: I've added different ways to display the dropdown content. 

- With Icon
- With button
- Without anything simply `li` element

File: src/components/ae-card/ae-card.md (hunk 2 above)
Comment (sadiqevani) [reply src/components/ae-card/ae-card.md]: Yep, my mistake.

File: src/components/ae-icon/ae-icon.md (hunk 1 above)
Comment (sadiqevani) [reply src/components/ae-icon/ae-icon.md]: Fixed.

File: src/components/ae-card/ae-card.md (hunk 3 above)
Comment (davidyuk) [reply src/components/ae-card/ae-card.md]: > I've added different ways to display the dropdown content.

please, do this in the documentation for **AeDropdown** component

**Pull request closed without merging**